internal/storage/v2/clickhouse/tracestore/reader.go

```diff
@@ -106,13 +116,23 @@ func (r *Reader) GetTraces(
 
 func scanSpanRow(rows driver.Rows) (dbmodel.Span, error) {
 	var (
-		span            dbmodel.Span
-		rawDuration     int64
-		eventNames      []string
-		eventTimestamps []time.Time
-		linkTraceIDs    []string
-		linkSpanIDs     []string
-		linkTraceStates []string
+		span                  dbmodel.Span
+		rawDuration           int64
+		boolAttributeKeys     []string
```

Comment:
Why do we need vars, can we not read directly into fields of the span struct?

Reply:
@yurishkuro We need vars here because each column of a nested has to be queried individually whereas the span struct groups the attributes' key value pairs together for convenience.